client/src/styles/components/_profile.scss

```diff
@@ -49,4 +49,10 @@
 		overflow-x: auto;
 	}
 
+	button {
```

Comment:
You can, but I would still define it as a class and put your styling for your button in the _buttons.scss